src/bindings/python/src/pyopenvino/core/async_infer_queue.cpp

```diff
@@ -229,54 +225,45 @@ void regclass_AsyncInferQueue(py::module m) {
             :rtype: bool
     )");
 
-    cls.def(
-        "wait_all",
-        [](AsyncInferQueue& self) {
-            return self.wait_all();
-        },
-        R"(
-        One of 'flow control' functions. Blocking call.
-        Waits for all InferRequests in a pool to finish scheduled work. 
+    cls.def("wait_all",
+            &AsyncInferQueue::wait_all,
+            R"(
+            One of 'flow control' functions. Blocking call.
+            Waits for all InferRequests in a pool to finish scheduled work.
 
-        Function releases GIL, other threads can work while this function waits.
-    )");
+            Function releases GIL, other threads can work while this function waits.
```

Comment:
```suggestion
            Function releases GIL, other threads can work while this function is running in the background.
```
I think this is a better overall statement. Please copy it to other docs as well.

Reply:
Updated.